boilerplates/react-union-boilerplate-liferay-basic/hero-portlet/src/main/java/eu/reactunion/boilerplate/portlet/data/InitData.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) Lundegaard, s.r.o. 2018 - All Rights Reserved
+ * Proprietary and confidential. Unauthorized copying of this file, via any medium is strictly prohibited.
+ */
+package eu.reactunion.boilerplate.portlet.data;
+
+/**
+ * @author Roman Srom (roman.srom@lundegaard.eu)
+ */
+public class InitData {
+
+    private Textation textation;
```

Comment:
Textation is not a word, use TextContent, Text, or something similar

Reply:
hero-portlet folder is old code. It was moved to liferay folder. "Textation" was used in react widget in the past.

Reply:
Sorry about that!